make/git/hook-pre-commit.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
On my box, /bin/sh -> /bin/dash, and I believe that this script uses a bash-ism. Not sure why I only saw this recently though. Should I change this back?

Reply:
Making it bash seems fine: we did the same thing with git_review.sh